docs/acp/user/developing-an-app/developing-an-app.md

```diff
@@ -28,130 +38,201 @@ You need to have a clear understanding of what your app API will provide to othe
 Before you start with the development, you should design your API schema files. Depending on your requirements, you can have an OpenAPI or an [Async API](#async-api) schema file. In this step, you define the [Sync API](#sync-api) endpoints your app will provide to others, the messages you will emit or consume, and the data you expect to work with.
```

Comment:
@VladislavStrelchenko , please check that you have anchors #sync-api and #async-api in this document. It looks like you have removed "##### Async API" section, so these links won't work anymore

Reply:
I will remove the liks